src/AtomId.ts

```diff
@@ -0,0 +1,28 @@
+import IndexMap from './IndexMap';
+
+export default class AtomId {
+  site: number;
+
+  index: number;
+
+  timestamp: number;
+
+  constructor(site: number, index: number, timestamp: number) {
+    this.site = site;
+    this.index = index;
+    this.timestamp = timestamp;
+  }
+
+  static compare(a: AtomId, b: AtomId): number {
+    if (a.timestamp === b.timestamp) return b.site - a.site;
```

Comment:
Please add some comments here explaining that:

- Older event < younger event (smaller timestamp < larger timestamp)
- Older site < younger site